mrp_brewing/models/company.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+
+from openerp import api, fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    warehousekeeper_nb = fields.Char(

```

Comment:
just wondering if nb is an appropriate shortname. @robinkeunen ok for you ?

Reply:
you are right, cambridge dictionary recommends _no_ usage

Reply:
done